test/screenshot/spec/mdc-select/classes/baseline-enhanced.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
```

Comment:
Under `classes/` you went with `...-enhanced` but under `mixins` you went with `enhanced-...`... pick one? I think just `enhanced.html` for this and `enhanced-disabled.html` for the disabled one could work.